bitnami/metrics-server/templates/pdb.yaml

```
@@ -2,14 +2,8 @@
 apiVersion: policy/v1beta1
 kind: PodDisruptionBudget
 metadata:
-  labels:
-    app: {{ template "metrics-server.name" . }}
-    chart: {{ template "metrics-server.chart" . }}
-    release: {{ .Release.Name }}
-    heritage: {{ .Release.Service }}
-  name: {{ template "metrics-server.fullname" . }}
-  namespace: {{ .Release.Namespace }}
-
+  name: {{ template "common.names.fullname" . }}
```

Could we use `include`?

```suggestion
  name: {{ include "common.names.fullname" . }}
```

---

Fixed